### ISSUE TITLE
Ubuntu compile error

### ISSUE THREAD

**Issue by bingoko** [Microsoft/cpprestsdk#669]
Ubuntu 16.04 LTS
Openssl version: 1.1.1

It seems that the openssl 1.1.1 is not compatible with CPPRESTSDK

**Comment by jwtowner** [comment]
Correct, cpprestsdk is only compatible with OpenSSL 1.0.2 LTS currently. I imagine that cpprestsdk will have to move to the next LTS when it gets created later this year or early next. Likely OpenSSL 1.1.2.
